lib/foreman_maintain/cli/packages_command.rb

```diff
@@ -60,12 +66,21 @@ def execute
       subcommand 'is-locked', 'Check if update of packages is allowed' do
         interactive_option(['assumeyes'])
         def execute
-          locked = ForemanMaintain.package_manager.versions_locked?
-          puts "Packages are#{locked ? '' : ' not'} locked"
-          exit_code = locked ? 0 : 1
-          exit exit_code
+          run_scenario_or_rescue do
+            locked = ForemanMaintain.package_manager.versions_locked?
+            puts "Packages are#{locked ? '' : ' not'} locked"
+            exit_code = locked ? 0 : 1
+            exit exit_code
+          end
         end
       end
+
+      def run_scenario_or_rescue
+        yield
+      rescue NotImplementedError
+        puts 'Command is not implemented for Debian based operating systems!'
```

Comment:
nothing in this file says they are not  implemented on *Debian* ;-)

Reply:
I am adding comments for not implemented commands

Reply:
ah, now it's starting to make sense in my head :D